src/components/EntityPage/Participant/Utils/Holder.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import {Container} from "../../../FileRepoSidebar/ui";
+import { Link } from 'react-router-dom';
+import styled from "react-emotion";
+import {withTheme} from "emotion-theming";
+
+//taken from secondaryNavMenu
+const MenuWrapper = styled('ul')`
+  ${({ theme }) => theme.secondaryNav}
+`;
+
+class Holder extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    //wrap the item in a one-element array if the item is not itself an array
+    if(!(props.children instanceof Array)) {
+      this.tabIDs = [props.children.props.label];
+      this.children = [props.children];
+    } else {
+      this.tabIDs = props.children.map( child => child.props.label );
+      this.children = props.children;
+    }
+
+    this.state = {activeTab: this.tabIDs[0]};
+  }
+
+  /**
+   * Makes the buttons for the tabs.
+   *
+   * @param holder This, as it is needed inside a nested class
+   * @returns {*|Array|*[]}
+   */
+  makeTabButtons(holder) {
+    return this.tabIDs.map(tabId => {
+
+      const isActive = tabId === this.state.activeTab;
+
+      //needs to be a class to use setState to do the hovering. Would do in CSS, but with emotion it's too complicated and hacky
```

Comment:
You can also use css at the same time.

see `src/components/CohortBuilder/VirtualStudiesMenu/index.js:36` and `src/components/CohortBuilder/VirtualStudiesMenu/index.postcss`. You can do the same with plain `.css` file if you prefer.

Just remember to use `className` and not `class` when setting a class in react.

Reply:
I will need to run it locally to see what can be done. I'll do that after you push.

Reply:
Okay! I'll Slack you when i push